# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/perl5/use_db3.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./ext/DB_File/config.in.orig	2003-07-07 22:12:50.000000000 -0400
+++ ./ext/DB_File/config.in	2003-07-07 22:16:44.000000000 -0400
@@ -9,18 +9,20 @@
 #    Change the path below to point to the directory where db.h is
 #    installed on your system.
 
-INCLUDE	= /usr/local/BerkeleyDB/include
+#INCLUDE	= /usr/local/BerkeleyDB/include
 #INCLUDE	= /usr/local/include
 #INCLUDE	= /usr/include
+INCLUDE	= /usr/include/db3
 
 # 2. Where is libdb?
 #
 #    Change the path below to point to the directory where libdb is
 #    installed on your system.
 
-LIB	= /usr/local/BerkeleyDB/lib
+#LIB	= /usr/local/BerkeleyDB/lib
 #LIB	= /usr/local/lib
 #LIB	= /usr/lib
+LIB	= /usr/lib
 
 # 3. What version of Berkely DB have you got?
 #
@@ -93,5 +95,6 @@
 #    you have picked.
  
 #DBNAME = -ldb-2.4.10
+DBNAME = -ldb-3
 
 # end of file config.in
